### ISSUE TITLE
[BUG] Broken link in documentation

### ISSUE THREAD

**Issue by oceanofthelost** [sierrafoxtrot/srecord#39]
Following man pages have a link to external PDF which no longer exists. 

1. [srec_ti_tagged_16(5)](https://linux.die.net/man/5/srec_ti_tagged_16)
1. [srec_ti_tagged(5)](https://linux.die.net/man/5/srec_ti_tagged)

Each man page above, the section "See Also" includes the following link:

<http://www.dataio.com/pdf/Manuals/Unifamily/981‐0014‐016.pdf>

Which when clicked returns a HTTP 404 error.

**Comment by oceanofthelost** [comment]
I can try and find alternative links for each case, unless someone has existing copies.

Should external references live under the etc folder or an external folder? This would remove the need for relying on an external site always being present.

**Comment by oceanofthelost** [comment]
Using Wayback Machine
1. [srec_emon52](https://web.archive.org/web/20020313021908/http://sbprojects.fol.nl/knowledge/fileformats/emon52.htm)
2. [srec_fpc](https://web.archive.org/web/20020620171514/http://sbprojects.fol.nl/knowledge/fileformats/fpc.htm)
3. [srec_signetics](https://web.archive.org/web/20020620171936/http://sbprojects.fol.nl/knowledge/fileformats/signetics.htm)

**Comment by marcows** [comment]
See also https://github.com/marcows/SRecord-deprecated/commit/2e8540a9422be16960349f64b2dca7abafc3403f for some website address fixes.

More than 8 years old, might be outdated partially.

**Comment by jtxa** [comment]
I would suggest Wayback machine, if no successor of that page exists.

The local `crc16-ccitt.html` file was retrieved from Wayback machine.
Should we do it also this way? Personally, I would remove that file and replace it by the link. No need to add different copyright things into the repository.

Perhaps add a github link checker action. Not on PRs, but cyclic once a month or so.
I don't know which one is good, this one was my first google hit: https://github.com/marketplace/actions/check-links-with-linkcheck

**Comment by oceanofthelost** [comment]
I would agree having some kind of link checker on a periodic basis would be beneficial to make sure links are still valid.
